### DISCUSSION TITLE
Un ripetitore zigbee marca X amplia anche per Y?

### DISCUSSION THREAD

**Discussion by Massimoflea** [indomus/forum#1702] [ZigBee, Z-Wave, Thread, Matter]
![3C37CE8D-D24A-460C-97FB-F5484379D18B](https://user-images.githubusercontent.com/106017106/169693775-62f14702-0a6d-42f0-b079-9890196e00fc.jpeg)

Ciao a tutti!! 
Ho un problema con un dispositivo zigbee connesso ed installato ad alexa direttamente . Posto al piano superiore di casa mi son reso conto che non prende il segnale e ho pensato a questo ripetitore. Vedendo però che il ripetitore si installa su tuya o smarthome non so se effettivamente possa fare al
Caso mio. Questo può  aiutarmi ad amplificare il segnale zigbee in generale e non solo per determinati dispositivi .?

**Comment by Robyxxx** [comment]
Ho più o meno lo stesso problema? Hai risolto con questo dispositivo?

**Comment by Massimoflea** [comment]
Si risolto Il giorno 4 ott 2022, alle ore 09:17, Robyxxx ***@***.***> ha scritto:﻿
Ho più o meno lo stesso problema? Hai risolto con questo dispositivo?

**Reply by Robyxxx** [reply]
In che modo si connette alla rete? E' sufficiente alimentarlo?
